website_product_brand/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+product-attribute
```

Comment:
Added this file in root.

Can you please explain how to add file with contents ?

Thanks.